docs/tools/hedgehog-engine/unleashed/files.md

```diff
@@ -0,0 +1,12 @@
+---
+description: File format tools for the Sonic Unleashed
+---
+
+## Unleashed Mod Toolbox
+
+Command line tool that allows the extraction and decompression of Sonic Unleashed .ar.00 and .arl files.
+
+The tool will not convert .ar.01, .ar.02 etc, however you can bypass this by changing the file extension to ar.00.
```

Comment:
Remove this remark, as it isn't needed. You just need to have every AR part file in the folder when you want to extract it.